src/Uno.UI/UI/Xaml/Controls/ItemsStackPanel/ItemsStackPanel.netstd.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
```

Comment:
File name should be `ItemsStackPanel.wasm.cs`.

Reply:
Will be fixed in separate PR.